core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -211,7 +211,10 @@ void handleUncaughtThrowable(Throwable t) {
   @CheckForNull
   private final ChannelTracer channelTracer;
   private final Channelz channelz;
+  @CheckForNull
   private Boolean zeroBackends; // a flag for doing channel tracing when flipped
```

Comment:
nit: sorry to bring this up again, but I just realized it might be even more clear if we made this `haveBackends` since the `zero` part is still effectively a negation.

Reply:
Done. Thanks @zpencer